### ISSUE TITLE
content structure

### ISSUE THREAD

**Issue by grudelsud** [devcon-london/devcon-london.github.io#6]
new site map

![website map](https://user-images.githubusercontent.com/310976/35435264-f7e42388-0281-11e8-9b16-7db95a84ba66.png)

**Comment by acewf** [comment]
considering if we could/should split members into categories ex:
- backend
- frontend

**Comment by andrevenancio** [comment]
yeah maybe tags would be nice. If I want to find someone who does webgl, I can filter members per technology, or tech (react, angular, etc)

**Comment by grudelsud** [comment]
that'd be cool, the problem I see is: who is going to curate the tags? where are they going to be stored? at the moment we only have a google spreadsheet with content entered by users at submission date (that could have happened 2-3 years ago!)

It somehow reminds me of [Devart](https://devart.withgoogle.com/), the project I did with google while at Stinkdigital, pulling content and tags directly from users Github repos. The problem (which makes it also interesting TBH) with this approach would be to have all members to create a fork of a template repo and follow some guidelines to write their own bio and tags. 

It'd be cool because we could somehow opensource a way of writing a mini bio on GH and would make clear that a user is a member because (s)he has forked the organisation's repo.
Let's think about it!

**Comment by acewf** [comment]
i like this idea, because it enables every member only to add is bio if they want, not forcing anyone is concerned with privacy to participate.
To feed the participants list with their repos links, we could have a JSON on this repo, where the participants can do a PR to add them.

**Comment by marcelkornblum** [comment]
It seems overly complex to me TBH. GH already allows people to create bios. And if we make all DevCon members into GH DevCon org members, there's inbuilt functionality for whether people want their membership to be public or not. 

In terms of skillset tags, why dont we scrape the repos on people's GH accounts for the tags on those repos, and pull out e.g. the most common 5?

**Comment by grudelsud** [comment]
> In terms of skillset tags, why dont we scrape the repos on people's GH accounts for the tags on those repos, and pull out e.g. the most common 5?

Re. tags, it sounds more like a useless complication to me at this stage. Is it what I'm doing? what I'm interested exploring? I might have tons of private repos with the companies I work(ed) with, I might not have anything of my current interests, etc.
